browser.go

```diff
@@ -310,6 +310,11 @@ func (b *Browser) run(ctx context.Context) {
 	}
 }
 
+// PID returns the browser process ID
+func (b *Browser) PID() int {
+	return b.process.Pid
```

Comment:
Please note that the process could be `nil` when the browser instance is created by `chromedp.RemoteAllocator`. As commented by mvdan in #566, maybe it's better to export the process field. And can you provide the example to show how it will be used (we can add it to the doc comment)?

Reply:
Agree, it makes sense, changed to expose Process, also added example to the doc comment